pydevices/acq400Devices/acq400_base.py

```diff
@@ -23,52 +23,49 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 
-import sys
 import threading
 import socket
 import time
-import inspect
-
-import numpy as np
+from queue import Queue, Empty
+import numpy
 import MDSplus
 
-if MDSplus.version.ispy3:
-    from queue import Queue, Empty
-else:
-    from Queue import Queue, Empty
+try:
```

Comment:
Dont import acq400_hapi here. import it as regular package
```
import acq400_hapi
```
 in the methods you use it.. acq400_hapi is a third party package just like pexpect, serial, etc. hence it should be placed in PYTHONPATH